docs/zh/client/other-language-client-user-guide.md

```diff
@@ -31,14 +31,23 @@
 ### 1.2.2 Http接口返回格式
 该Http接口返回的是JSON格式、UTF-8编码，包含了对应namespace中所有的配置项。
 
-返回内容Sample如下：
```

Comment:
Please update the English version of the document as well.

Reply:
Thanks for notice, updated.